lib/tests/streamlit/runtime/media_file_manager_test.py

```diff
@@ -372,10 +372,10 @@ def test_remove_orphaned_files_multiple_sessions(self, mock_get_session_id):
         )
 
 
-NUM_THREADS = 15
-
-
 class MediaFileManagerThreadingTest(unittest.TestCase):
+    # The number of threads to run our tests on
+    NUM_THREADS = 50
+
```

Comment:
Essentially unrelated to this PR, but I moved the `NUM_THREADS` definition inside `MediaFileManagerThreadingTest` (and increased it a bit) to keep parity with `UploadedFileManagerThreadingTest`